molgenis-python/src/main/resources/templates/molgenis.py.ftl

```diff
@@ -118,25 +118,42 @@ class Session():
             data = json.dumps({"entities" : entities}))
         if response.status_code == 201:
             return [resource["href"].split("/")[-1] for resource in response.json()["resources"]]
-        response.raise_for_status();
+        self._raise_for_status(response);
         return response;
 
+    def update(self, entity_name, row_id, data = {}, **kwargs):
+        '''Updates 1 or more attributes of a single entity row
+
+        Args:
+        entity -- fully qualified name of the entity
+        row_id -- id of the row of the entity to update
+        data -- dictionary mapping attribute name to non-file attribute value for the entity row, gets merged with the kwargs argument
+        **kwargs -- keyword arguments get merged with the data argument
+        '''
+        server_response_list = []
+        for key in data:
+            response = self.session.put(self.url+'v1/'+quote_plus(entity_name)+'/'+str(row_id)+'/'+key, data='"'+data[key]+'"', headers = self._get_token_header_with_content_type())
```

Comment:
So if one of the updates fails, the ones that got succesfully updated remain updated and the others are not attempted?